dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -773,12 +776,78 @@ internal class RumViewScopeTest {
         }
         verifyNoMoreInteractions(mockWriter)
         assertThat(result).isSameAs(testedScope)
+        assertThat(testedScope.pendingErrorCount).isEqualTo(pending - 1)
+    }
+
+    @Test
+    fun `𝕄 send event 𝕎 handleEvent(ErrorSent) on active view {isCrash = true}`(
+        @LongForgery(1) pending: Long
+    ) {
+        // Given
+        fakeEvent = RumRawEvent.ErrorSent(testedScope.viewId, isCrash = true)
+        testedScope.pendingErrorCount = pending
+
+        // When
+        val result = testedScope.handleEvent(fakeEvent, mockWriter)
+
+        // Then
+        argumentCaptor<RumEvent> {
+            verify(mockWriter).write(capture())
+            assertThat(lastValue)
+                .hasAttributes(fakeAttributes)
+                .hasUserExtraAttributes(fakeUserInfo.additionalProperties)
+                .hasViewData {
+                    hasTimestamp(fakeEventTime.timestamp)
+                    hasName(fakeName)
+                    hasUrl(fakeUrl)
+                    hasDurationGreaterThan(1)
+                    hasVersion(2)
+                    hasErrorCount(1)
+                    hasCrashCount(1)
+                    hasResourceCount(0)
+                    hasActionCount(0)
+                    hasLongTaskCount(0)
+                    isActive(true)
+                    hasNoCustomTimings()
+                    hasUserInfo(fakeUserInfo)
+                    hasViewId(testedScope.viewId)
+                    hasApplicationId(fakeParentContext.applicationId)
+                    hasSessionId(fakeParentContext.sessionId)
+                }
+        }
+        verifyNoMoreInteractions(mockWriter)
+        assertThat(result).isSameAs(testedScope)
+        assertThat(testedScope.pendingErrorCount).isEqualTo(pending - 1)
+    }
+
+    @Test
+    fun `𝕄 do nothing 𝕎 handleEvent(ErrorSent) on active view {unknown viewId}`(
+        @Forgery viewUuid: UUID,
+        @BoolForgery isCrash: Boolean,
+        @LongForgery(1) pending: Long
+    ) {
+        // Given
+        val viewId = viewUuid.toString()
+        assumeTrue(viewId != testedScope.viewId)
+        fakeEvent = RumRawEvent.ErrorSent(viewId, isCrash)
+        testedScope.pendingErrorCount = pending
+
+        // When
+        val result = testedScope.handleEvent(fakeEvent, mockWriter)
+
+        // Then
+        verifyZeroInteractions(mockWriter)
+        assertThat(result).isSameAs(testedScope)
+        assertThat(testedScope.pendingErrorCount).isEqualTo(pending)
     }
 
     @Test
-    fun `𝕄 send event 𝕎 handleEvent(SentResource) on active view`() {
+    fun `𝕄 send event 𝕎 handleEvent(ResourceSent) on active view`(
```

Comment:
you do not want to assert also the `mockWriter`  as you do for the ErrorEvent above ?